google/model-viewer
# Add a poster feature to match with modelviewer.dev/example/tester

Add a create / display poster feature.

Tested workflow: 

1. Upload a model.
2. Click `create poster` in pencil tab, user will see a poster in the output snippet.
3. Click `display poster` in pencil tab, user will see the poster previewed in model-viewer-preview
4. Upload another model, the previous poster settings should be cleared.
5. Upload a snippet with poster (with reveal="interaction" potentially), user should see the poster before model loaded (or interacted with poster)
